var/spack/repos/builtin/packages/atlas/package.py

```diff
@@ -54,7 +54,13 @@ class Atlas(Package):
             url='http://sourceforge.net/projects/math-atlas/files/Developer%20%28unstable%29/3.11.34/atlas3.11.34.tar.bz2')
 
     variant('shared', default=True, description='Builds shared library')
-    variant('pthread', default=False, description='Use multithreaded libraries')
+
+    variant(
+        'multithreading', default='none',
```

Comment:
alternative is to call this `threading`. Intel link advisor words this as

> Select threading layer:

Reply:
@alalazo btw, do you prefer to call this `multithreading=none` or `threading= sequential`?